src/ensembl/src/content/app/browser/track-panel/track-panel-list/TrackPanelListItem.scss

```diff
@@ -21,7 +21,7 @@
   }
 
   &:hover,
-  &.currentDrawerView {
+  &.highlightTrack {
```

Comment:
How about changing `highlightTrack` class name to `trackHighlighted` instead? Also I think we'll need to rename `listItem` class at the top to be `track`.

Reply:
That's done